packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -44,7 +44,8 @@ declare module "react" {
   }
 }
 
-const HIDE_ZOOM_CONTROLS_DELAY = 2000;
+const SHOW_ZOOM_CONTROLS_DELAY_MS = 100;
+const HIDE_ZOOM_CONTROLS_DELAY_MS = 400;
```

Comment:
IMO it could be 100 for both of them. This delay when you move out seems weird.

Reply:
Fair point but I think HIDE_ZOOM_CONTROLS_DELAY_MS should be a bit longer then SHOW_ZOOM_CONTROLS_DELAY_MS just to give more time for recognising the current zoom level state. SHOW_ZOOM_CONTROLS_DELAY_MS with 200ms seems better.